src/osdag/cad/common_logic.py

```diff
@@ -2050,6 +2054,12 @@ def display_3DModel(self, component, bgcolor):
             self.col = self.module_class()
             self.ColObj = self.createColumnInFrameCAD()
 
+            if self.component == "Model":
+                osdag_display_shape(self.display, self.ColObj, update=True)
+        elif self.mainmodule == 'Struts in Trusses':
+            self.S = self.module_class()
+            #self.SObj = self.createColumnInFrameCAD()
+
             # if self.component == "Model":
             #     osdag_display_shape(self.display, self.ColObj, update=True)
```

Comment:
Is this comment needed?

Reply:
Again for the CAD part of the Strut Module, it should have a self.SObj = something and not ideally self.createColumnInFrameCAD(), Parth asked to replicate the changes for Column here as well, but then since I later found out that it was not directly causing an error/playing a part in report generation, I commented it instead.

Reply:
OK. Ideally add a comment for why it's needed the next time (I won't insist for this one for this rebase).